src/scandipwa/app/component/GoogleTagManager/utils/Product.js

```diff
@@ -106,21 +116,29 @@ export default class Product {
         return {};
     }
 
+    /**
+     * @param categories
+     * @returns {string}
+     */
     static getCategory(categories = []) {
-        const { url_path = '' } = categories.slice(-1)[0] || {};
-        return url_path;
+        return categories.slice(0, 5).map(({ name }) => name).join('/');
     }
 
+    /**
+     * @param variant
+     * @param type_id
+     * @returns {number}
+     */
     static getPrice(variant, type_id) {
         const {
-            price: {
-                minimalPrice: {
-                    amount: {
+            price_range: {
+                minimum_price: {
+                    final_price: {
                         value: discountValue = null
                     } = {}
                 } = {},
-                regularPrice: {
-                    amount: {
+                regular_price: {
+                    final_price: {
```

Comment:
Are you 100% sure these changes are not project-specific? It seems they might be.